File: Pea Soup and Pancakes/peasoup.py
q = int(input())
exists = False
for _ in range(q):
    peas = False
    pancakes = False
    menus = int(input())
    name = input()
    for _ in range(menus):
        food = input()
        if food == 'pea soup':
            peas = True
        elif food == 'pancakes':
            pancakes = True
    if peas and pancakes:
        print(name)
        exists = True
        break
if not exists:
    print('Anywhere is fine I guess')

File: Pea Soup and Pancakes/peasoup.js
var restaurants = parseInt(readline()); // the number of restaurants
let exists = false;
for (let i = 0; i < restaurants; i++) {
    let menus = parseInt(readline()); // the number of menus
    let name = readline();
    let peas = false;
    let pancakes = false;

    for (let j = 0; j < menus; j++) {
        let food = readline();
        if (food === "pea soup") {
            peas = true;
        } else if (food === "pancakes") {
            pancakes = true;
        }
    }

    if (peas && pancakes) {
        console.log(name);
        exists = true;
        break;
    }
}

if (!exists) {
    console.log("Anywhere is fine I guess");
}